src/Spryker/Zed/EventBehavior/Persistence/Propel/Schema/spy_event_behavior.schema.xml

```diff
@@ -2,7 +2,7 @@
 <database xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="zed" xsi:noNamespaceSchemaLocation="http://static.spryker.com/schema-01.xsd" namespace="Orm\Zed\EventBehavior\Persistence" package="src.Orm.Zed.EventBehavior.Persistence">
 
     <table name="spy_event_behavior_entity_change">
-        <column name="id_event_behavior_entity_change" type="integer" autoIncrement="true" primaryKey="true"/>
+        <column name="id_event_behavior_entity_change" type="BIGINT" autoIncrement="true" primaryKey="true"/>
```

Comment:
This is probably good enough for now. But you should probably consider reseting sequence. Or have custom sequences which are not depended on data insert/delete.